package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "aqovia-vscode-extension-pack",
+  "displayName": "Aqovia VSCode ExtensionPack",
+  "description": "Aqovia Visual Studio Code Extension Pack",
+  "version": "1.3.39",
+  "publisher": "Aqovia",
+  "license": "MIT",
+  "engines": {
+    "vscode": "^1.80.0"
+  },
+  "scripts": {
+    "pack": "vsce package --out VSCodeExtensionsPack.vsix"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Aqovia/VisualStudioCode.ExtensionPack"
+  },
+  "categories": [
+    "Extension Packs"
+  ],
+  "extensionPack": [
+    "42crunch.vscode-openapi",
+    "adisreyaj.swagger-snippets",
+    "analytic-signal.preview-pdf",
+    "arjun.swagger-viewer",
+    "bierner.lit-html",
+    "corylulu.csharp-interpolated-string-converter",
+    "christian-kohler.npm-intellisense",
+    "DavidAnson.vscode-markdownlint",
+    "dbaeumer.vscode-eslint",
+    "donjayamanne.git-extension-pack",
+    "DotJoshJohnson.xml",
+    "eamodio.gitlens",
+    "eriklynd.json-tools",
+    "formulahendry.dotnet-test-explorer",
+    "Fudge.auto-using",
+    "GitHub.copilot",
+    "hbenl.vscode-test-explorer",
+    "hnw.vscode-auto-open-markdown-preview",
+    "Jenkey2011.string-highlight",
+    "leizongmin.node-module-intellisense",
+    "LittleFoxTeam.vscode-python-test-adapter",
+    "mads-hartmann.bash-ide-vscode",
+    "mermade.openapi-lint",
+    "monotykamary.inline-yaml",
```

Comment:
Does this do anything that `redhat.vscode-yaml` doesn't?

Reply:
I've tried disabling it on my machine, and I can't see any difference so I think the redhat extension is taking precedence anyway. Removed the extension.